### PR TITLE
[Docs] Update Japanese translation of feature_backlight.md.

### PR DESCRIPTION
**Pull request by shelaf**
## Description

Japanese documentation update.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [X] Documentation

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

tag: @mtei

### CODE REVIEW

**Comment by mtei** [comment]
@umi-umi @s-show @hsgw @satt99 @takashiski @yoshihara よかったらレビューお願いします。

File: docs/ja/feature_backlight.md [hunk 1]
```diff
@@ -1,253 +1,224 @@
...
+| `BL_STEP` | バックライトレベルを循環する         |
+| `BL_ON`   | バックライトを最大輝度に設定する     |
+| `BL_OFF`  | バックライトをオフにする             |
+| `BL_INC`  | バックライトレベルを上げる           |
+| `BL_DEC`  | バックライトレベルを下げる           |
+| `BL_BRTG` | バックライトの明滅動作を切り替える   |
+
+## 関数群 :id=functions
+
+これらの関数を使って、カスタムコードでバックライトを変更することができます:
```
Comment (mtei) [inline docs/ja/feature_backlight.md:36]: 英語の辞書引くと、this/these には、「(後に続く内容をさして)次の, 次に述べる」という意味／訳もありうるので、

```suggestion
次の関数を使って、カスタムコードでバックライトを変更することができます:
```
あるいは、
```suggestion
以下の関数を使って、カスタムコードでバックライトを変更することができます:
```

File: docs/ja/feature_backlight.md [hunk 2]
```diff
@@ -1,253 +1,224 @@
...
 
-有効にするには、rules.mk に以下を追加します:
+上記ドライバのいずれもキーボードに適用されていない場合(例えば、バックライトを制御するのに別の IC を使用している場合)、QMK が提供しているこの簡単な API を使ってカスタムバックライトドライバを実装することができます。有効にするには、`rules.mk` に以下を追加します:
 
 ```makefile
 BACKLIGHT_DRIVER = custom
 ```
 
-カスタムドライバ API を実装する場合、提供されるキーボードフックは以下の通りです:
+それからこれらのフックのいずれかを実装します:
```
Comment (mtei) [inline docs/ja/feature_backlight.md:199]:
```suggestion
それから次のフックのいずれかを実装します:
```
あるいは、
```suggestion
それから以下のフックのいずれかを実装します:
```

File: docs/ja/feature_backlight.md [hunk 3]
```diff
@@ -1,253 +1,224 @@
...
-  original document: 0.9.10:docs/feature_backlight.md
-  git diff 0.9.10 HEAD -- docs/feature_backlight.md | cat
+  original document: 0.9.44:docs/feature_backlight.md
+  git diff 0.9.44 HEAD -- docs/feature_backlight.md | cat
 -->
 
 多くのキーボードは、キースイッチを貫通して配置されたり、キースイッチの下に配置された個々の LED によって、バックライトキーをサポートします。この機能は通常スイッチごとに単一の色しか使用できないため、[RGB アンダーグロー](ja/feature_rgblight.md)および [RGB マトリックス](ja/feature_rgb_matrix.md)機能のどちらとも異なりますが、キーボードに複数の異なる単一色の LED を取り付けることは当然可能です。
 
-QMK は *パルス幅変調*(*Pulse Width Modulation*) すなわち PWM として知られている技術で急速にオンおよびオフを切り替えることで、これらの LED の輝度を制御できます。PWM 信号のデューティサイクルを変えることで、調光の錯覚を起こすことができます。
+QMK は *パルス幅変調* (*Pulse Width Modulation*) すなわち PWM として知られている技術で、特定の比率で急速にオンおよびオフを切り替えることで、これらの LED の輝度を制御できます。PWM 信号のデューティサイクルを変えることで、調光の錯覚を起こすことができます。
```
[the remark this answers is not in the document]
Comment (shelaf) [reply docs/ja/feature_backlight.md]: 修正しておきました

File: docs/ja/feature_backlight.md (hunk 1 above)
Comment (shelaf) [reply docs/ja/feature_backlight.md]: 「次」の方を採用してみました

**Comment by mtei** [comment]
I think this is ready to merge.

**Comment by noroadsleft** [comment]
ありがとうございました！

**Pull request merged**